amplify/backend/function/completeEvent/src/index.js

```diff
@@ -0,0 +1,94 @@
+/* Amplify Params - DO NOT EDIT
+	API_HOUSEHOLDAPP_EVENTHANDLERTABLE_ARN
+	API_HOUSEHOLDAPP_EVENTHANDLERTABLE_NAME
+	API_HOUSEHOLDAPP_EVENTTABLE_ARN
+	API_HOUSEHOLDAPP_EVENTTABLE_NAME
+	API_HOUSEHOLDAPP_GRAPHQLAPIIDOUTPUT
+	ENV
+	REGION
+Amplify Params - DO NOT EDIT */
+
+const AWS = require('aws-sdk');
+const ddb = new AWS.DynamoDB.DocumentClient();
+
+/**
+ * @type {import('@types/aws-lambda').APIGatewayProxyHandler}
+ */
+exports.handler = async (eventHandler) => {
```

Comment:
I see you had to change the Lambda function argument from the default "event" to eventHandler to get it to work

Reply:
Since `completeEvent` is internal and will only be called by `completeTask` then we can fully control what the input will be. In this case, `completeTask` will always send it an eventHandler.

Maybe a better name for this function would have been `completeTaskEvent`

Reply:
Yeah I think that name would make more sense. I understand what you mean though.